tests/test_views.py

```diff
@@ -0,0 +1,27 @@
+from django.urls import reverse
+from django.test import modify_settings, TestCase
+
+from .factories import ArticleFactory
+
+
+class TestView(TestCase):
+    slug_name = 'ellobiidae'
+
+    @modify_settings(INSTALLED_APPS={'append': ['tests']})
+    def test_index_response_ok(self):
+        response = self.client.get(reverse('article_index'))
+        self.assertEqual(response.status_code, 200)
+
+    @modify_settings(INSTALLED_APPS={'append': ['tests']})
+    def test_detail_response_ok(self):
```

Comment:
I suspect further tests may benefit from `tests` being in the `INSTALLED_APPS`. Perhaps it makes sense to remove it in this final test rather than add it for the others?